packages/esm-patient-tests-app/src/test-results/panel-view/panel.component.test.tsx

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import LabSetPanel from './panel.component';
+import { isDesktop, useLayoutType, type LayoutType } from '@openmrs/esm-framework';
+import { type ObsRecord } from '../../types';
+import { type OBSERVATION_INTERPRETATION } from '@openmrs/esm-patient-common-lib';
+
+jest.mock('@openmrs/esm-framework', () => ({
+  formatDate: jest.fn().mockReturnValue('January 1, 2024'),
+  isDesktop: jest.fn().mockReturnValue(true),
+  useLayoutType: jest.fn().mockReturnValue('desktop'),
+}));
+
+const mockUseLayoutType = jest.mocked(useLayoutType);
+const mockIsDesktop = jest.mocked(isDesktop);
+
+const mockConceptMeta = {
+  display: '',
+  hiNormal: 0,
+  hiAbsolute: 0,
+  hiCritical: 0,
+  lowNormal: 0,
+  lowAbsolute: 0,
+  lowCritical: 0,
+  units: 'g/dL',
+  range: '12-16',
+  getInterpretation: function (value: string): OBSERVATION_INTERPRETATION {
+    return 'NORMAL';
+  },
+};
+
+const mockBasePanel: ObsRecord = {
+  resourceType: 'Observation',
+  id: 'test-id',
+  conceptUuid: 'test-uuid',
+  category: [
+    {
+      coding: [
+        {
+          system: 'test-system',
+          code: 'test-code',
+          display: 'Laboratory',
+        },
+      ],
+    },
+  ],
+  code: {
+    coding: [
+      {
+        code: 'test-code',
+        display: 'Test Display',
+      },
+    ],
+    text: 'Test Text',
+  },
+  effectiveDateTime: '2024-01-01T10:00:00Z',
+  issued: '2024-01-01T10:00:00Z',
+  name: 'Complete Blood Count',
+  value: '120',
+  interpretation: 'NORMAL',
+  relatedObs: [],
+  meta: mockConceptMeta,
+  referenceRange: [],
+};
```

Comment:
If these mocks are meant to be used in more than one place, it'd be better to move them to the top-level `__mocks__` directory.